user/reference/overview.md

```diff
@@ -32,7 +32,7 @@ An [OS X](/user/reference/osx/) environment for Objective-C and other OS X speci
 
 ### Windows
 
-A [Windows](/user/reference/windows/) environment running Windows Server 2016.
+A [Windows](/user/reference/windows/) environment running Windows Server 1803.
```

Comment:
would it make sense to link to the release docs? https://docs.microsoft.com/en-us/windows-server/get-started/whats-new-in-windows-server-1803

Reply:
Windows names it "Windows Server, version 1803"

Reply:
:100: